src/Runner.Common/JobServerQueue.cs

```diff
@@ -476,8 +471,27 @@ private async Task ProcessSummaryUploadQueueAsync(bool runOnce = false)
                         }
                         catch (Exception ex)
                         {
-                            Trace.Info("Catch exception during summary file upload to results, keep going since the process is best effort.");
-                            Trace.Error(ex);
+                            try
+                            {
+
+                                var issue = new Issue() { Type = IssueType.Warning, Message = $"Caught exception during summary file upload to results. {ex.Message}" };
```

Comment:
Do we want to make this warning shows up on the user's face?
You might need to do some server work to hide it if we don't want.

Reply:
Definitely.  Server side PR on separate thread.